api/internal/localizer/localizer.go

```diff
@@ -127,3 +134,43 @@ func (lc *Localizer) localizeFile(path string) (string, error) {
 	}
 	return locPath, nil
 }
+
+// localizeDir localizes root path and returns the localized path
+func (lc *Localizer) localizeDir(path string) (string, error) {
+	ldr, err := lc.ldr.New(path)
+	if err != nil {
+		return "", errors.Wrap(err)
+	}
+	defer func() { _ = ldr.Cleanup() }()
+
+	var locPath string
+	if repo := ldr.Repo(); repo != "" {
+		// TODO(annasong): check if able to add localize directory
+		locPath = locRootPath(path, filesys.ConfirmedDir(repo), filesys.ConfirmedDir(ldr.Root()))
```

Comment:
IIUC this branch gets hit when the path was remote, but has already been cloned to a tempdir somewhere. So we do not expect the directory to be relative to the localizer's root (as in the other branch), and if we dig through the code we know it was previously made a `ConfirmedDir`. Still, I'm hesitant to simply convert the strings to ConfirmedDir like this without actually doing the confirmation--it means we're making assumptions that the current behaviour will be maintained. Is it even important to this method that the dir has been confirmed?

Reply:
Yes, `locRootPath` needs the `repo` to be a confirmed directory because it wants to create a localized path without symlinks. 

I can actually call `ConfirmDir`. 

Not for this PR, but what do you think about changing `ifc.Loader`'s `Root()` method to return a `filesys.ConfirmedDir` instead? It shouldn't break implementations of `ifc.Loader` for testing because filesystems in memory can also have `filesys.ConfirmedDir`, right?

Reply:
That idea makes sense to me, and offhand I _feel_ like it should work fine.